pointcloud/octree/basic_test.go

```diff
@@ -0,0 +1,252 @@
+package octree
```

Comment:
I would like to see a test where you load a test PCD file, and turn the loaded pointcloud into a basicOctTree, and then check if 
1. The number of points are what you expect
2. And the location and metadata of the points are what you expect

Reply:
Will add a test as soon as can.

Reply:
Added